docs/data/material/components/snackbars/snackbars.md

```diff
@@ -9,109 +9,91 @@ waiAria: https://www.w3.org/TR/wai-aria-1.1/#alert
 
 # Snackbar
 
-<p class="description">Snackbars provide brief notifications. The component is also known as a toast.</p>
-
-Snackbars inform users of a process that an app has performed or will perform. They appear temporarily, towards the bottom of the screen. They shouldn't interrupt the user experience, and they don't require user input to disappear.
-
-Snackbars contain a single line of text directly related to the operation performed.
-They may contain a text action, but no icons. You can use them to display notifications.
+<p class="description">Also known as a toast, snackbars are used for brief notifications of a process that has been performed or will perform.</p>
 
 {{"component": "modules/components/ComponentLinkHeader.js"}}
 
-**Frequency**: Only one snackbar may be displayed at a time.
+## Basics
 
-## Simple snackbars
+```jsx
+import Snackbar from '@mui/material/Snackbar';
+```
 
-A basic snackbar that aims to reproduce Google Keep's snackbar behavior.
+Inspired by Google Keep, the demo below shows a basic snackbar with a text element and two actions.
 
 {{"demo": "SimpleSnackbar.js"}}
 
 ## Customization
 
-Here are some examples of customizing the component.
-You can learn more about this in the [overrides documentation page](/material-ui/customization/how-to-customize/).
-
-{{"demo": "CustomizedSnackbars.js"}}
-
-## Positioned snackbars
+### Position
 
-In wide layouts, snackbars can be left-aligned or center-aligned if they are consistently placed on the same spot at the bottom of the screen, however there may be circumstances where the placement of the snackbar needs to be more flexible.
-You can control the position of the snackbar by specifying the `anchorOrigin` prop.
+Use the `anchorOrigin` prop to control the snackbar position on the screen.
 
 {{"demo": "PositionedSnackbar.js"}}
 
-## Message Length
+### Content
 
-Some snackbars with varying message length.
+```jsx
+import SnackbarContent from '@mui/material/SnackbarContent';
+```
 
-{{"demo": "LongTextSnackbar.js"}}
+Use the Snackbar Content component for adding text and actions to the snackbar.
 
-## Transitions
+{{"demo": "LongTextSnackbar.js"}}
 
-### Consecutive Snackbars
+### Usage with Alerts
```

Comment:
I'm cool with either term as long as apply it consistently. As for capitalization, I'd keep "Alerts" capitalized since it refers to our component specifically (rather than the general concept of _an_ alert).